### ISSUE TITLE
MSSQL Auto increment primary key

### ISSUE THREAD

**Issue by Shimakuchi** [TablePlus/TablePlus#295]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
**Microsoft SQL Server 2016**

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
**Version 1.0 (90)**

3. The steps to reproduce this issue:

Create a table like so:

```SQL
SET ANSI_NULLS ON
GO
SET QUOTED_IDENTIFIER ON
GO
CREATE TABLE  [mytablename](
	[id] [int] IDENTITY(1,1) NOT NULL,
	[lf_user_id] [int] NOT NULL,
	[username] [varchar](36) NOT NULL,
	[password] [varchar](36) NOT NULL,
	[valid] [bit] NULL,
	[created_at] [datetime2](6) NOT NULL,
	[updated_at] [datetime2](6) NOT NULL,
	[locked] [bit] NULL,
 CONSTRAINT [pk_mytablename] PRIMARY KEY CLUSTERED 
(
	[id] ASC
)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY],
 CONSTRAINT [uq_mytablename] UNIQUE NONCLUSTERED 
(
	[lf_user_id] ASC,
	[username] ASC
)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
) ON [PRIMARY]

GO
```

- Insert a new Row `CMD+ALT+I`
- Fill all the Fields and try to commit the changes

I get the error
`DEFAULT or NULL are not allowed as explicit identity values.
All DML changes were reverted`

---
Insert via script works fine
```sql
USE [MYDATABASE]
GO
INSERT INTO [mydatabase].[mytablename]
           ([lf_user_id]
           ,[username]
           ,[password]
           ,[valid]
           ,[created_at]
           ,[updated_at]
           ,[locked])
     VALUES
           (0000007
           ,'Max Mustermann'
           ,'DUMMY'
           ,1
           ,getutcdate()
           ,getutcdate()
           ,0)
GO
```

**Comment by huyphams** [comment]
I fixed: #295 #294 #293 on this build: https://www.dropbox.com/s/3orqil53ktk8yzk/TablePlus.zip?dl=0
Please let me know if it fixes your issues.

**Comment by Shimakuchi** [comment]
YES, works great

- [x] #295
- [x] #294 
- [x] #293 

THANKS
